medical_physician/wizard/medical_physician_unavailable.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# © 2015 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalPhysicianUnavailableWizard(models.TransientModel):
+    _name = 'medical.physician.unavailable.wizard'
+    _description = 'Medical physicians unavailable wizard'
+    physician_id = fields.Many2one(
+        'medical.physician', 'Physician', required=True
+        )
+    date_start = fields.Datetime(
+        string='Start', default=fields.date.today, required=True
+        )
+    date_end = fields.Datetime(
+        string='End', default=fields.date.today, required=True
```

Comment:
Always use lambda for default to allow overrides. Also, you're mixing types - need to use Datetime.now, not Date.today:

```
default=lambda s: fields.Datetime.now()
```